fe/fe-core/pom.xml

```diff
@@ -828,15 +828,15 @@ under the License.
                                     <arg value="--java_out=${basedir}/target/generated-sources/proto"/>
                                     <arg value="${starrocks.home}/gensrc/proto/lake_service.proto"/>
                                 </exec>
-                                <exec executable="python">
+                                <exec executable="${PYTHON}">
```

Comment:
suggest make env PYTHON as a variable

Reply:
like this `<exe exec executable="${PYTHON}">` ?

Reply:
you can ref thrift.

```
<profile>
  python3 profile:
<profile>
```

Reply:
please review the latest commit.